Mail2Bug/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="CsQuery" version="1.3.4" targetFramework="net45" />
```

Comment:
AngleSharp seems to be the better-maintained library at this point. After implementing with csquery, I got AngleSharp mostly working but ran into some text encoding issues (e.g. fancy quotes weren't rendered correctly). My inclination is leave csquery in use barring functional issues but am open to discussion.